pkg/datasource/datasource.go

```diff
@@ -83,7 +85,25 @@ func NewOSS(ctx context.Context, c client.Client, endpoint *v1alpha1.Endpoint, o
 	return &OSS{Client: mc}, nil
 }
 
-// TODO: implement Check with specific `options`
-func (oss *OSS) Check(ctx context.Context, options any) error {
+func (oss *OSS) Check(ctx context.Context, ossInfo *v1alpha1.OSS) error {
+	if ossInfo.Bucket != "" {
```

Comment:
@zqq454224016  Great! I did some changes on the datasource definition. Please rebase your code to the latest code once this pr #157  merged

Reply:
ok